cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -17,6 +17,9 @@ set(CMAKE_SYSTEM_VERSION 1)
 
 set(CMAKE_CROSSCOMPILING TRUE)
 
+# Don't do compiler autodetection by default, but leave option for frameworks that require it
+option(EMSCRIPTEN_FORCE_COMPILERS "Force C/C++ compiler" ON)
```

Comment:
Let's move this option and the comment down close to the place of use, line 96? Looks a bit odd to decouple this up there and the use down below.